### ISSUE TITLE
Data leak for nearby songkick events

### ISSUE THREAD

**Issue by matkoniecz** [clementine-player/Clementine#7107]
#4898  #4993 were wontfixed despite Clementine making unwanted, unexpected data to remote service and leaking user data.

Even when such data is not needed by user who disabled it.

Please, remove this data leak.

**Comment by bughunter2** [comment]
Agreed. This really should be an opt-in preference which should be disabled by default.
